test/e2e_node/e2e_node_suite_test.go

```diff
@@ -45,8 +45,6 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework/testfiles"
 	"k8s.io/kubernetes/test/e2e/generated"
 	"k8s.io/kubernetes/test/e2e_node/services"
-
```

Comment:
Removing this blank line sepetating groups of imports requires the imports to be sorted, which fails gofmt.

We should leave this line.